.slashrc

```diff
@@ -224,6 +226,9 @@ class MediaPlugin(slash.plugins.PluginInterface):
     from lib.platform import info
     return info()["os"]
 
+  def _get_component(self):
+    return sys.argv[2].split("/")[1]
+  
```

Comment:
sys.argv does not always contain the component.  Users can run all tests by only specifying  `run test/` directory, for example